src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -44,13 +44,6 @@ def dependency(self):
     def path(self):
         return self._dependency.path
 
-    def register_library(self, name: str) -> MaybeGraph:
```

Comment:
@ericvergnaud : For now, shall we stick with the `import`, not have the `library`, and add it to the `DependencyGraph` right away without requiring an `import ...`

Reply:
mmm... I suspect that would only increase confusion when discovering an already complex framework...